Docs/models docs for dummies (#1013)
deepmipt/DeepPavlov: docs/intro/quick_start.rst

:doc:`Telegram Bot doc page </integrations/telegram>` and others in the
Integrations section for more info.

Pretrained models
~~~~~~~~~~~~~~~~~

> DeepPavlov provides wide range of pretrained models and skills.

Reviewer: ```suggestion
DeepPavlov provides a wide range of pretrained models and skills.
```

See :doc:`features overview </features/overview>` for more info. Please
note that most of our models are trained on specific datasets for
specific tasks and may require further training on you data.

> List of our out-of-the-box models you can find below.

Reviewer: ```suggestion
You can find a list of our out-of-the-box models `below <#out-of-the-box-pretrained-models>`_.
```

Docker images
~~~~~~~~~~~~~

You can run DeepPavlov models in :doc:`riseapi </integrations/rest_api>` mode
via Docker without installing DP. Both your CPU and GPU (we support NVIDIA graphic
processors) can be utilised, please refer our `CPU <https://hub.docker.com/r/deeppavlov/base-cpu>`_
and `GPU <https://hub.docker.com/r/deeppavlov/base-gpu>`_ Docker images run instructions.

Out-of-the-box pretrained models
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> While the best way to solve most of the NLP tasks lies through collecting dataset

Reviewer: ```suggestion
While the best way to solve most of the NLP tasks lies through collecting datasets
```

> and training model according to the domain and actually task itself, DeepPavlov can

Reviewer: ```suggestion
and training models according to the domain and an actual task itself, DeepPavlov can
```

Insult Detection
================

Insult detection predicts whether a text (e.g, post or speech in some
public discussion) is considered insulting to one of the persons it is
related to.

.. table::
    :widths: auto

    +----------+------------------------------------------------------------------------------------------------+-------------------------------------------+
    | Language | DeepPavlov config                                                                              | Demo                                      |
    +==========+================================================================================================+===========================================+
    | En       | :config:`insults_kaggle_conv_bert <classifiers/insults_kaggle_conv_bert.json>`                 | https://demo.deeppavlov.ai/#/en/insult    |
    +----------+------------------------------------------------------------------------------------------------+-------------------------------------------+

Sentiment Analysis
==================

> Classify text according prevailing emotion (positive, negative) in it.

Reviewer: ```suggestion
Classify text according prevailing emotion (positive, negative, etc.) in it.
```
